index.html

```diff
@@ -13,7 +14,66 @@
   <![endif]-->
 </head>
 
-<body>
-<!-- Use your own code or past solution for Great Idea Web Page HTML here! -->
-</body>
-</html>
\ No newline at end of file
+  <body>
```

Comment:
this is a MASSIVE block of code. No white space, no comments to break up/explain things. Very hard on the eyes of any other developers viewing your code. Work on spacing it out a bit and using comments if necessary.